engine-tests/src/test/resources/logback-test.xml

```diff
@@ -1,23 +1,25 @@
-<!-- Copyright 2021 The Terasology Foundation -->
-<!-- SPDX-License-Identifier: Apache-2.0 -->
```

Comment:
do we not need the copyright and license info in here?

Reply:
imho it's not worth the noise for config files.

can't say I have any precedent to cite on that though.

Reply:
okay, fair enough. if we at some point notice we do want/need it after all, I guess we can just add it.